addons/hr_holidays/static/src/scss/accrual_plan_level.scss

```diff
@@ -55,63 +50,42 @@ $o-hr-holidays-border-color: map-get($grays, '300');
 
             // Whole record
             .o_hr_holidays_body {
-                margin-left: 8px;
-                padding-top: 20px;
-
                 // Left side 'Level'
                 .o_hr_holidays_timeline {
                     @include o-position-absolute($top: 32px, $left: 6px);
-                    width: 90px;
-                    padding: 3px 0px;
-                    border-radius: 3px;
-                    background-color: $o-gray-200;
-                    box-shadow: 0 1px 2px rgba(0,0,0,.1);
+                    width: var(--o_hr_holidays_timeline-width);
                 }
 
                 // Actual kanban card
                 .o_hr_holidays_card {
-                    position: relative;
-                    margin-left: 22px;
-                    margin-right: 2px;
                     width: 500px;
-                    border-radius: 3px;
-                    background-color: $o-gray-200;
-                    box-shadow: 0 1px 2px rgba(0,0,0,.1);
 
                     // Triangle
                     &:before {
                         content: '';
-                        @include o-position-absolute($top: 12px, $left: -17px);
+                        @include o-position-absolute($top: 12px, $left: -18px);
                         margin-left: 10px;
                         width: 14px;
                         height: 14px;
```

Comment:
commit 771dc529383f66446aa307baada638a0c9ce192c

You've removed the arbitrary values for the circle but not for the triangle, I think it would make it easier to use css variables here as well. For example your top positioning of the triangle should be the same as the top positioning of the circle to have perfect alignment (here 2 pixels are missing). The same variable could be used to position the `o_hr_holidays_timeline` vertically aligned with the circle.